### DISCUSSION TITLE
Custom Rename App and or Change Icon?

### DISCUSSION THREAD

**Discussion by FTP-ForthePeople** [Authenticator-Extension/Authenticator#684] [Ideas]
Appreciate the app. Thanks for making it!

For security purposes, Is it possible to customize how the app shows up in chrome. If my laptop was stolen for example, if someone could go to the app extension and see authenticator they could get in to my accounts pretty easily. If its renamed and the icon is changed it is harder to identify for a user who doesn't know what they are looking for.

I'm not a programmer by any means, but I did go into the mainfest.json and try and fool around. I changed( "default_title": "***", "name": "***", "short_name": "***") but when I rebooted didn't see any change and the app still worked. Didn't try the icon changing because I believe I'd have to change the sfv file. At that point I was happy I hadn't broken the ext already.

Thanks for any info/help you can provide.

**Comment by Sneezry** [comment]
> If my laptop was stolen for example, if someone could go to the app extension and see authenticator they could get in to my accounts pretty easily.

You can set a password in security menu, and nobody can access your secret without knowing the password.

**Reply by FTP-ForthePeople** [reply]
Are you referring to the security menu in chrome? I checked and set my security to enhanced protection, but didn't see an option for a password.

Are you suggesting something like this?
[Set password for your browser](https://chrome.google.com/webstore/detail/set-password-for-your-bro/cjmjgijhapgicbhmniemjkjeaedanank/related?hl=en)

**Reply by Sneezry** [reply]
It's the extension settings menu

![image](https://user-images.githubusercontent.com/1621293/121444743-a096ae00-c9c2-11eb-85b0-c74f9be6b752.png)
